tests/test_models.py

```diff
@@ -98,6 +98,10 @@ def test_model_singlepsr_noise_chrom_nondiag(nodmx_psrs,caplog):
     m=models.model_singlepsr_noise(nodmx_psrs[1], dm_var=True,
                                    dm_type=None, chrom_gp=True,
                                    chrom_gp_kernel='nondiag')
+    assert log10_sigma = True
```

Comment:
This assigns the value `True` to the variable named `log10_sigma`.  I think you want to test some property of the model.  What is log10_sigma?  Where does it appear in the model?  How can check that it's in the right place?